### PR TITLE
Feat/spinner

### PR DESCRIPTION
**Pull request by dpellier**

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-storybook#463.0 ✘  
* 
  * build ✘

**Review by ady642 — APPROVED** [review]

File: packages/stencil/components/spinner/src/components/osds-spinner/styles/osds-spinner.typography.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@import '~@ovhcloud/ods-theming/typography/ods-theming-typography';
+@import '~@ovhcloud/ods-theming/typography/ods-component-typography';
+@import './osds-spinner.mixins';
+
+// Main CSS mixin for typography
+@mixin osds-spinner-theme-typography() {
+  :host([size='md']) .spinner__wrapper {
+    //$typography-properties: ods-get-typography-properties(body, '400');
```
Comment (aesteves60) [inline packages/stencil/components/spinner/src/components/osds-spinner/styles/osds-spinner.typography.scss:8]: remove comment

File: packages/stencil/components/tooltip/.stencil/screenshot-cache.json
Comment (astagnol) [inline packages/stencil/components/tooltip/.stencil/screenshot-cache.json:1]: to rm

File: packages/stencil/components/tooltip/.stencil/screenshot_2c7f5908e8b0b1224fdc5bfc060e12d0.png.js
Comment (astagnol) [inline packages/stencil/components/tooltip/.stencil/screenshot_2c7f5908e8b0b1224fdc5bfc060e12d0.png.js:1]: to rm

File: packages/stencil/components/tooltip/.stencil/screenshot_587b10a226753f0044741157cbe71dac.png.js
Comment (astagnol) [inline packages/stencil/components/tooltip/.stencil/screenshot_587b10a226753f0044741157cbe71dac.png.js:1]: to rm

**Review by astagnol — COMMENTED** [review]
Files related to Tooltip should be removed (screenshots & some .ts files)

Screenshot tests diff each time we relaunch them. Not sure the animation to none is working here

**Comment by dpellier** [comment]
> Screenshot tests diff each time we relaunch them. Not sure the animation to none is working here

Should be fixed now, tried 5 times in a row locally without issues

**Comment by ovh-cds** [comment]
CDS Report build-storybook#551.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build-storybook#551.2 ✘  
* 
  * build ✘

**Review by astagnol — APPROVED** [review]

**Pull request merged**